src/legacy/core_plugins/metrics/server/lib/vis_data/get_annotations.js

```diff
@@ -28,10 +29,11 @@ function validAnnotation(annotation) {
     !annotation.hidden;
 }
 
-export async function getAnnotations(req, panel, esQueryConfig, searchStrategy, capabilities) {
+export async function getAnnotations(req, panel, esQueryConfig, searchStrategy, capabilities, series) {
```

Comment:
Please do small changes in the arguments order
getAnnotations(req, panel, `series`, esQueryConfig, searchStrategy, capabilities) {

Reply:
Destructring has been applied